### DISCUSSION TITLE
ユーザーの電話番号保存方法について

### DISCUSSION THREAD

**Discussion by Keitaro0226** [lit-iphone-course/lit-iphone-course#38] [Question-original] [answered]
(1) 実現したいこと
今、ユーザーデータをfirebaseから取得しようとしているのですが、ユーザーのdocumentIDを電話番号で保存しています。アプリ起動時のデータ取得にユーザーの電話番号が必要なのですが、authを実装していないのでどうやって電話番号を保持しようか迷っています。
(2) 実現するために自分が試した内容とその結果
authを実装するのは現実的ではないのでUserDefaultsに電話番号のみ保存する方法がいいかなと思っています。

(3) 調べた内容や資料
<img width="1331" alt="スクリーンショット 2022-09-22 16 39 56" src="https://user-images.githubusercontent.com/52875782/191687255-35fc7ceb-5312-4f5c-9fd8-2a9a08e2ced6.png">

他に良い方法があれば教えていただきたいです！！

[github](https://github.com/Keitaro0226/MorningCallApp)

**Comment by txxxxc** [comment, marked as answer]
Firestoreでやれなくもないですけど、UserDefaultsで事足りるなら全然UserDefaultsで大丈夫だと思います！！

**Reply by Keitaro0226** [reply]
ありがとうございます！UserDefaultsで実装しようと思います！
